src/style.css

```
@@ -0,0 +1,113 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: rgba(6, 11, 53, 1);
```

Do you really need rgba here? Use rgb or hex
```suggestion
  color: rgb(6, 11, 53);
```